src/lib/core/say.h

```diff
@@ -63,6 +63,13 @@ enum say_level {
 	say_level_MAX
 };
 
+/** Log color red */
+#define ANSI_COLOR_RED "\x1b[31m"
+/** Log color yellow */
+#define ANSI_COLOR_YELLOW "\x1b[33m"
+/** Log color reset */
+#define ANSI_COLOR_RESET "\x1b[0m"
+
```

Comment:
No reason to move that to `.h` file and exposing these defines to whoever includes this file. Let's move that to `.c` file.

Reply:
moved